tokens/style-dictionary.config.js

```diff
@@ -11,53 +11,135 @@
  * governing permissions and limitations under the License.
  */
 
-const path = require("path");
+import { dirname, join, sep } from "path";
```

Comment:
This was rewritten as ESM instead of CJS to support Style Dictionary's breaking change to v4